paymentsheet/src/main/java/com/stripe/android/paymentelement/embedded/form/EmbeddedFormInteractorFactory.kt

```diff
@@ -52,9 +54,21 @@ internal class EmbeddedFormInteractorFactory @Inject constructor(
             onFormCompleted = { eventReporter.onPaymentMethodFormCompleted(PaymentMethod.Type.USBankAccount.code) },
         )
 
+        val formType = formHelper.formTypeForCode(paymentMethodCode)
+        val formArguments = formHelper.createFormArguments(paymentMethodCode)
+        if (formType is FormHelper.FormType.MandateOnly) {
+            embeddedSelectionHolder.set(
```

Comment:
This is my least favorite part of this change. The selection didn't get set properly when the form was already complete. Which is what this solves. I'm going to try to follow this up with a better approach.